src/Halcyon/Model.php

```diff
@@ -980,12 +983,12 @@ public function delete()
      */
     protected function performDeleteOnModel()
     {
-        $this->newQuery()->delete($this->fileName);
+        $this->newQuery()->delete();
```

Comment:
Why was this changed?

Reply:
The delete method on the model doesn't accept any parameters.